registry/registry_test.go

```diff
@@ -803,8 +803,7 @@ func TestValidRemoteName(t *testing.T) {
 
 		"_docker/_docker",
 
-		// Disallow consecutive underscores and periods.
-		"dock__er/docker",
```

Comment:
My preference with these cases is to move from invalid to valid to make the behavior change more explicit